src/utils/array.js

```diff
@@ -0,0 +1,10 @@
+export const range = (a, b) => [...Array(b + 1).keys()].slice(a);
+
+export const partition = (arr, predicate) =>
```

Comment:
Should we add tests for `partition` and `range`?

Reply:
Thats a good call. I'll make sure to add those.

Reply:
Addressed in c0cbc1c